esx_service/vmdk_ops.py

```diff
@@ -419,6 +423,7 @@ def set_policy_to_vmdk(vmdk_path, opts, vol_name=None):
                             vol_name=vol_name)
 
         if clean_err:
+            logging.warning("Failed to clean %s file", vmdk_path)
```

Comment:
Better to log the error_info which will provide more information about why it fails.